internal/fields/validate_test.go

```diff
@@ -237,6 +237,34 @@ func Test_parseElementValue(t *testing.T) {
 			},
 			fail: true,
 		},
+		// arrays of objects can be stored in groups, even if not recommended
+		{
+			key: "host",
+			value: []interface{}{
+				map[string]interface{}{
+					"id":       "somehost-id",
+					"hostname": "somehost",
+				},
+				map[string]interface{}{
+					"id":       "otherhost-id",
+					"hostname": "otherhost",
+				},
+			},
+			definition: FieldDefinition{
```

Comment:
I'm wondering if there isn't any parser like this in the ECS project? I have a feeling that we're reimplementing it.